tests/test_scale_properties.py

```diff
@@ -1,66 +1,94 @@
-"Contains a collection of pytest tests that validates the scale properties."
+"Contains collection of pytest tests that validates the scale properties."
+
+import pytest
 
 from nidaqmx.constants import UnitsPreScaled
 from nidaqmx.errors import DaqError
 from nidaqmx.scale import Scale
 
 
-class TestScaleProperty(object):
-    """Contains a collection of pytest tests that validates the scale properties."""
+@pytest.mark.parametrize("persisted_scale", ["double_gain_scale"], indirect=True)
+def test__scale__float64_property__returns_default_value(persisted_scale):
+    """Test for validating getter for float property."""
+    scale = persisted_scale.load()
+    default_value = scale.lin_slope
+
+    assert scale.lin_slope == default_value
+
+
+@pytest.mark.parametrize("persisted_scale", ["double_gain_scale"], indirect=True)
+def test__scale__set_float64_property__returns_assigned_value(persisted_scale):
+    """Test for validating setter for float property."""
+    scale = persisted_scale.load()
+
+    scale.lin_slope = 5
+
+    assert scale.lin_slope == 5
+
+
+@pytest.mark.parametrize("persisted_scale", ["polynomial_scale"], indirect=True)
+def test__scale__float64_list_property__returns_default_value(persisted_scale):
+    """Test for validating getter for float list property."""
+    scale = persisted_scale.load()
+    default_value = scale.poly_forward_coeff
+
+    assert scale.poly_forward_coeff == default_value
+
+
+@pytest.mark.parametrize("persisted_scale", ["polynomial_scale"], indirect=True)
+def test__scale__set_float64_list_property__returns_assigned_value(persisted_scale):
+    """Test for validating setter for float list property."""
+    scale = persisted_scale.load()
+
+    coeff_list = [1.0, 2.0]
+    scale.poly_forward_coeff = coeff_list
+
+    assert scale.poly_forward_coeff == coeff_list
+
+
+def test__scale__get_Invalid_float64_list_property__throws_DaqError():
```

Comment:
Naming: use lowercase snake_case.

Optional: I think it's slightly clearer if you use the given and when clauses to describe what is invalid, like "given a linear scale, when you get a poly scale property, it throws DaqError." i.e. `test__linear_scale__get_poly_scale_property__throws_daqerror`. I'm assuming we aren't trying to test the error case for each data type. However, your proposed function name is okay too (except the capitalization).

Reply:
Changed the function name to lower snake_case and specified the 'given' in name.